app/src/main/java/ar/com/wolox/android/example/utils/Constants.java

```diff
@@ -0,0 +1,12 @@
+package ar.com.wolox.android.example.utils;
+
+/**
+ * File to store constants
+ */
+public final class Constants {
+
+    private Constants() {
+    }
+
+    public static final String WOLOX_WEB = "http://www.wolox.com.ar";
```

Comment:
You dont need this. Only set the String on the strings.xml resource. If you need a constant try to create it only in the object that needs that (If you dont need to share with other objects)

Reply:
Ok, I removed it :+1: